lib/routific.rb

```diff
@@ -11,11 +11,13 @@
 # Main class of this gem
 class Routific
   attr_reader :token, :visits, :fleet, :options
+  @@token = nil
 
   # Constructor
   # token: Access token for Routific API
-  def initialize(token)
-    @token = token
+  def initialize()
+    Routific.validate_token
+    @token = Routific.token
```

Comment:
Assigning `@token` seems unnecessary.

Reply:
That's true, I think I must have kept in there thinking that we would allow instances to have distinct tokens from the class, but we decided against that. Let me remove it!